s3torchbenchmarking/utils/generate_datasets_files.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Do we have a preference between writing bash vs. Python script? I'd prefer the latter, simply because it's easier to reason about (and less error-prone, from experience)